Wikibase/Sniffs/Commenting/DisallowedDocTagsSniff.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Custom sniff that reports deprecated PHPDoc tags (e.g. known misspellings) and replaces them with
+ * their preferred counterparts.
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+class Wikibase_Sniffs_Commenting_DisallowedDocTagsSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * @var array Set a value to null if a tag is forbidden, but no automatic fix should be made.
```

Comment:
This needs to document which of key and value is the incorrect and which is the correct value, precisely because of the confusing inconsistency which makes it hard to infer this from the array value :)

Reply:
I updated the documentation. Hope this helps.

Reply:
Thanks, much better IMO :) immediately understandable now.